dll/win32/shell32/wine/shellord.c

```diff
@@ -2314,6 +2314,20 @@ INT WINAPI SHHandleUpdateImage(PCIDLIST_ABSOLUTE pidlExtra)
 
 BOOL WINAPI SHObjectProperties(HWND hwnd, DWORD dwType, LPCWSTR szObject, LPCWSTR szPage)
 {
+    LPITEMIDLIST pidl = NULL;
+    switch (dwType)
+    {
+        case SHOP_FILEPATH: pidl = ILCreateFromPathW(szObject); break;
```

Comment:
```suggestion
        case SHOP_FILEPATH:
            pidl = ILCreateFromPathW(szObject);
            break;
```

Reply:
Does it really look better to you?

The additional types would just be `case SHOP_PRINTER: pidl = ILCreatePrinterPidl(name); break;` etc. But printers are a long way down on the shell to do list.

Reply:
just our usual general code style.